src/forms/medication-dispense-review.component.tsx

```diff
@@ -305,7 +305,11 @@ const MedicationDispenseReview: React.FC<MedicationDispenseReviewProps> = ({
           }
           min={0}
           max={config.dispenseBehavior.restrictTotalQuantityDispensed ? quantityRemaining : undefined}
-          value={medicationDispense.quantity.value}
+          value={
+            quantityRemaining <= medicationDispense.quantity.value
+              ? quantityRemaining
+              : medicationDispense.quantity.value
+          }
```

Comment:
Do we need this here, because I believe it is incorrect... it doesn't allowing setting the dispense value to be greater than the quantity remaining, *even if* the ".restrictTotalQuantityDispensed" property is set to false.  Can we just keep this as-is and rely on the validation to flag whether the number types by the user is valid?